src/Rules/Exceptions/CatchWithUnthrownExceptionRule.php

```diff
@@ -32,6 +40,14 @@ public function processNode(Node $node, Scope $scope): array
 			];
 		}
 
+		$isCheckedException = TrinaryLogic::createFromBoolean($this->reportUncheckedExceptionDeadCatch)->lazyOr(
```

Comment:
This code isn't very readable to me. We want to early-return only if `$this->reportUncheckedExceptionDeadCatch` is false. So please do all this logic inside `if (!$this->reportUncheckedExceptionDeadCatch) { ... } `.

Thank you.